### PR TITLE
fix(server): Add deis event messages to application log stream

### LINKED ISSUE #65: filter out messages that are coming from the logger component
Right now we cannot do the filtering needed within fluentd so we should filter out all messages from the logger component within the component itself.

### PR DESCRIPTION
**Pull request by jchauncey**
fixes #65

### CODE REVIEW

**Comment by bacongobbler** [comment]
LGTM:

```
><> deis create
Creating Application... done, created zigzag-landmass
Git remote deis added
remote available at ssh://git@deis.10.245.1.3.xip.io:2222/zigzag-landmass.git
><> deis logs
INFO [zigzag-landmass]: bacongobbler created initial release
INFO [zigzag-landmass]: domain zigzag-landmass added
```

File: syslogish/server.go [hunk 1]
```diff
@@ -13,9 +13,10 @@ import (
 )
 
 const (
-	bindHost  = "0.0.0.0"
-	bindPort  = 1514
-	queueSize = 500
+	bindHost          = "0.0.0.0"
+	bindPort          = 1514
+	queueSize         = 500
+	controllerPattern = "^(INFO|WARN|DEBUG|ERROR)\\s+(\\[(\\S+)\\])+:(.*)"
```
Comment (mboersma) [inline syslogish/server.go]: Go regexes read better as [raw string literals](https://golang.org/ref/spec#String_literals):
```go
controllerPattern = `^(INFO|WARN|DEBUG|ERROR)\s+(\[(\S+)\])+:(.*)`
```

**Comment by mboersma** [comment]
Logging looks very Deis v1-like now, which is great:
```console
$ deis logs
INFO [quoted-underdog]: matt created initial release
INFO [quoted-underdog]: domain quoted-underdog added
INFO [quoted-underdog]: matt deployed 779ca3d
INFO [quoted-underdog]: Cleaning up RCS for releases older than v2 (latest)
INFO [quoted-underdog]: Cleaning up orphaned environment var secrets
quoted-underdog-v2-web-2n3ke -- 2016/04/23 17:11:58 Successfully copied home/quoted-underdog:git-779ca3d7/push/slug.tgz to slug.tgz
quoted-underdog-v2-web-2n3ke -- 2016/04/23 17:11:58 listening on 5000...
quoted-underdog-v2-web-2n3ke -- 2016/04/23 17:12:36 192.168.64.1 GET /
quoted-underdog-v2-web-2n3ke -- 2016/04/23 17:12:36 192.168.64.1 GET /favicon.ico
quoted-underdog-v2-web-2n3ke -- 2016/04/23 17:12:49 192.168.64.1 GET /
quoted-underdog-v2-web-2n3ke -- 2016/04/23 17:12:50 192.168.64.1 GET /favicon.ico
quoted-underdog-v2-web-2n3ke -- 2016/04/23 17:12:56 192.168.64.1 GET /
quoted-underdog-v2-web-2n3ke -- 2016/04/23 17:12:56 192.168.64.1 GET /favicon.ico
quoted-underdog-v2-web-2n3ke -- 2016/04/23 17:13:03 192.168.64.1 GET /
quoted-underdog-v2-web-2n3ke -- 2016/04/23 17:13:03 192.168.64.1 GET /favicon.ico
quoted-underdog-v2-web-2n3ke -- 2016/04/23 17:13:07 192.168.64.1 GET /
quoted-underdog-v2-web-2n3ke -- 2016/04/23 17:13:07 192.168.64.1 GET /favicon.ico
```
LGTM3, although the regex change I noted above would be nice.

**Comment by bacongobbler** [comment]
Would be nice to slap on a timestamp, but we got application event logs so I'm happy :)

File: syslogish/server.go (hunk 1 above)
Comment (jchauncey) [inline syslogish/server.go]: Ah OK I wasn't sure how to do that. I'll get the PR updated thanks

**Comment by jchauncey** [comment]
@bacongobbler k I'll see what I can do

**Comment by bacongobbler** [comment]
my LGTM still stands after the changes. :ship: :it:

**Pull request merged**